jcommune-view/jcommune-web-view/src/main/webapp/resources/javascript/app/dialog.js

```diff
@@ -322,7 +322,8 @@ $(function () {
         var tabNavigation = function (selectors) {
             $.each(selectors, function (idx, v) {
                 var func = function (e) {
-                    if ((e.keyCode || e.charCode) === tabCode) {
+                    //The "jQuery.which" property does not work on the "onkeypress" event in Firefox.
+                    if ((e.which || e.keyCode) === tabCode) {
                         e.preventDefault();
```

Comment:
I assume this comment from the commit message should be placed here:
>in internet explorer (specifically ie8) when you press "Enter" button on the item "input", event of "submit" doesn't work constantly, and this element loses focus. So in the "submitDialog" i call the appropriate method manually.

But I don't understand what "doesn't work constantly" means, could you please rephrase a bit and place an inline comment here.